source/frontend/src/features/mapSideBar/acquisition/tabs/stakeholders/detail/stakeholderOrganizer.ts

```diff
@@ -0,0 +1,84 @@
+import { Api_AcquisitionFile } from '@/models/api/AcquisitionFile';
```

Comment:
This looks a bit like unrelated tech-debt. I prefer this, but feel like this should have been done in a separate PR at least.

Reply:
True, I refactored to leverage similar props being passed. It also contains the grouping logic instead of being part of the view